docs/whats-new.asciidoc

```diff
@@ -1,17 +1,95 @@
+
+* Phishing with a spearphishing attachment, where an attacker drops an initial payload via a malicious document or shortcut
+* Signed binary proxy execution, a comprise of commonly abused living off the land (LotL) binaries and scripts
+* Privilege escalation via UAC bypass, which allows adversaries to bypass Windows User Account Control to elevate privileges
+
+These new capabilities leverage the malicious behavior preventions introduced in {stack} version 7.15 to stop various techniques for phishing, credential theft, living off the land (LotL), and achieving advanced persistence.
+
+*New integrations*
+
+{elastic-sec} ships the following new https://docs.elastic.co/en/integrations[integrations] in 7.16:
+
+* AWS WAF
+* Cisco Duo
+* GitHub
+* 1Password
+
+*ECS support for Osquery Manager integration*
+
+Host inspection is streamlined via the Osquery Manager integration for {agent}. Users can now {kibana-ref}/osquery.html#osquery-map-fields[map saved query results] directly to ECS to return normalized data.
+
+*Trusted applications enhancements*
+
+You can now configure <<trusted-apps-ov, trusted applications>> per policy instead of universally, allowing you to configure hosts that may require unique settings. This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise license]. In addition, when configuring trusted applications, you can use the `matches` operator to include wildcards in file paths.
+
+[role="screenshot"]
+image::whats-new/images/trusted-app.png[]
+
+*Add host isolation exceptions*
+
+You can add <<host-isolation-exceptions, host isolation exceptions>> for specific IP addresses that isolated hosts are still allowed to communicate with, even when blocked from the rest of your network.
+
+[role="screenshot"]
+image::whats-new/images/host-isolation-exception.png[]
+
+*Detection rule enhancements*
+
+* When <<import-export-rules-ui, exporting or importing>> detection rules, actions are now included.
+* The default query for <<create-indicator-rule, indicator match rules>> has been updated to check the indicator index patterns for matched indicators that have occurred within the past 30 days.
+
+*Easily view events that can be analyzed in the Alerts table*
+
+Alerts that can be opened in the visual event analyzer are denoted by the Analyze event button.
+
+[role="screenshot"]
+image::whats-new/images/analyze-event.png[]
+
+This option now appears as a visible inline action, whereas before it was only accessible via the overflow menu.
+
+[role="screenshot"]
+image::whats-new/images/alerts-page.gif[]
+
+*New ServiceNow integrations*
+
+This release delivers https://store.servicenow.com/sn_appstore_store.do#!/store/integrations[certified applications] for ServiceNow Security Operations (SecOps), ServiceNow IT Service Management (ITSM), plus a new connector for ServiceNow IT Operations Management (ITOM).
```

Comment:
```suggestion
This release delivers https://store.servicenow.com/sn_appstore_store.do#!/store/integrations[certified applications] for ServiceNow Security Operations (SecOps) and ServiceNow IT Service Management (ITSM), plus a new connector for ServiceNow IT Operations Management (ITOM).
``` 
Since the "plus" distinguishes the third item from the first two items, the first two items come across as a list of their own (of certified integrations)